corrector.rb

```diff
@@ -0,0 +1,6 @@
+class Corrector
+  def correct_name(name)
+    name.capitalize!
+    name[0...10] if name.length > 10
```

Comment:
### **Note**

- [ ] you can skip the if statement here because you already specified the length in the array[] and also you can make the number[0, 10]